### PR TITLE
Adding ion position cout

### PR DESCRIPTION
**Pull request by kadupitiya**
@jadhao:
Adding ion position cout.
Please have a look.

### CODE REVIEW

File: src/functions.cpp [hunk 1]
```diff
@@ -211,8 +213,16 @@ void make_movie(int num, vector<PARTICLE> &ion, INTERFACE &nanoparticle) {
                 type = "-1";
```
Comment (jadhao) [inline src/functions.cpp]: @kadupitiya I was wondering if we should correct something here now. for some viewers, may be even ovito, -1 as a type may be a little hard to read -- should we call this type = "2". basically type 1 is positively charged ion, type 2 is negatively charged ion. type 2 is more accurate than type -1.

File: src/functions.cpp [hunk 2]
```diff
@@ -211,8 +213,16 @@ void make_movie(int num, vector<PARTICLE> &ion, INTERFACE &nanoparticle) {
...
             outdump << setw(6) << i << "\t" << type << "\t" << setw(8) << ion[i].posvec.x << "\t" << setw(8)
                     << ion[i].posvec.y << "\t" << setw(8) << ion[i].posvec.z << endl;
+            if (!cpmdremote.verbose)
+                ions_pos_str =
+                        ions_pos_str + std::to_string(i) + "," + type + "," + std::to_string(ion[i].posvec.x) +
+                        "," + std::to_string(ion[i].posvec.y) + "," + std::to_string(ion[i].posvec.z) + "\n";
         }
         outdump.close();
+        ions_pos_str = ions_pos_str + "####_Ions_Position_Wrapper__Over_####";
```
Comment (jadhao) [inline src/functions.cpp]: @kadupitiya remove the extra _ in " "

File: src/functions.cpp (hunk 1 above)
Comment (jadhao) [reply src/functions.cpp]: unless of course, this type = -1 is used elsewhere in an important way and would need many other changes. i am ok with type = -1, but would prefer type 2 if that is a trivial change.

**Comment by kadupitiya** [comment]
@jadhao:
I think it is only used for ovito thing.
Maybe we should discuss this when we meet.
I am not using "type" anywhere.

**Pull request merged**